.github/workflows/register_repos.yaml

```diff
@@ -0,0 +1,54 @@
+name: Register repository
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  register-repository:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Register new repository
+        env:
+          GH_USERNAME: ${{ secrets.GH_USERNAME }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          python -m pip install requests ruamel.yaml aiohttp
+          python .github/workflows/register_repositories.py
+
+      - name: Capture diff output
+        id: diff
+        shell: bash
+        run: |
+          git diff config.yaml > config.diff
+
+      - name: Pushing up-to-date config file
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "GitHub Action"
+          git add config.yaml && git commit -m "[skip ci] Update configuration file."
+          # git push -f --set-upstream origin master
```

Comment:
Do we want to create a dedicated branch for the configuration file?

Reply:
I don't think there is a need for a separate branch.

Reply:
👍